src/interfaces/providerjson/providerjson.schema.json

```diff
@@ -55,6 +55,14 @@
                         "additionalProperties": false,
                         "description": "Security scheme for api key authorization.",
                         "properties": {
+                            "bodyType": {
+                                "$id": "ApiKeyBodyType",
+                                "description": "Body type to inject security value to.",
+                                "enum": [
+                                    "json"
+                                ],
+                                "type": "string"
+                            },
```

Comment:
Some dark magic happening here. Compared to `ApiKeyPlacement` if I use `$ref` on the field, it will be still created in place but with id and ref, so there is circular reference. And I am unable to convince `typescript-json-schema` library to generate `ApiKeyBodyType` in definitions. 🤷